demisto_sdk/commands/format/format_module.py

```diff
@@ -45,6 +47,7 @@
     'reputation': IndicatorTypeJSONFormat,
     'layout': LayoutBaseFormat,
     'layoutscontainer': LayoutBaseFormat,
+    'pre-process-rules': PreProcessRulesBaseFormat,
```

Comment:
why is it a "base" format?
```suggestion
    'pre-process-rule': PreProcessRulesBaseFormat,
```

Reply:
https://github.com/demisto/demisto-sdk/commit/685dbf70e06e21c5930de6d898461f08a82d7165